src/rexi/src/rexi.erl

```diff
@@ -213,7 +213,8 @@ stream(Msg, Limit, Timeout) ->
 
 %% @equiv stream2(Msg, 10, 300000)
 stream2(Msg) ->
-    stream2(Msg, 10, 300000).
+    Limit = config:get_integer("rexi", "stream_limit", 10),
+    stream2(Msg, Limit, 300000).
```

Comment:
Minor nit, but it might be prudent to go `stream/1` --> `stream/2` --> `stream/3` rather than `stream/1` --> `stream/3` on the off chance the timeout param in `stream/2` is converted from a hardcoded value to a config lookup like you did here (we also shouldn't be duplicating magic numbers).

Reply:
I see what you mean and I agree on not repeating constants.